assets/i18n/en.json

```
@@ -92,6 +92,9 @@
     "I18N_ANDROID_PAGE_UPDATES_SUBMIT_BUTTON_TEXT": "Notify Me",
     "I18N_ANDROID_PAGE_UPDATES_SUBTEXT": "And want to hear when the app is available<br>in your language? <span class=\"oppia-android-bold-text\">We'll notify you!</span>",
     "I18N_ANDROID_PAGE_VERSIONS_SUPPORTED_TEXT": "We support all Android versions going back to Lollipop (Android 5).",
+    "I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_1": "You're headed in the right direction, but you need to recheck your spelling.",
+	"I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_2": "You’re close to the right answer. Could you please correct your spelling?",
+	"I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_3": "Please recheck your spelling.",
     "I18N_ASSIGNED_STORIES_AND_SKILLS": "Assigned Stories and Skills",
```

Remove the extra indentation.

```suggestion
    "I18N_ANDROID_PAGE_VERSIONS_SUPPORTED_TEXT": "We support all Android versions going back to Lollipop (Android 5).",
    "I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_1": "You're headed in the right direction, but you need to recheck your spelling.",
    "I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_2": "You’re close to the right answer. Could you please correct your spelling?",
    "I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_3": "Please recheck your spelling.",
    "I18N_ASSIGNED_STORIES_AND_SKILLS": "Assigned Stories and Skills",
```

---

Done.